labs/coding-101-rest-basics-ga/6.md

```diff
@@ -40,7 +40,7 @@ To get the entire list of hosts, your request will be like this:
 * **URL** - `https://{APIC-EM-Server}/api/v1/host`
 	* Insert the IP address or hostname of the APIC-EM controller.
 * **Headers**
-	* For authentication, add the service ticket that was created earlier. On the left side of the header, add the text 'X-Auth-Token'. On the right side, add the service ticket value.
+	* For authentication, add the service ticket that was created earlier. On the left side of the header, add the text 'X-Auth-Token'. On the right side, add the service ticket value. See screenshot on next page for information on how to set this up in Postman.
 	* When using GET, a content header is not required. However, it's a good practice to specify the content in the header anyway as  'Content-Type' 'application/json'
```

Comment:
While you're in this file, should this next bullet point have a period at the end of it?